pkg/internal/tests/cross-tests/impl/diff.go

```diff
@@ -19,6 +19,8 @@ type PulumiDiffResp struct {
 type DiffResult struct {
 	TFDiff     tfcheck.TFChange
 	PulumiDiff PulumiDiffResp
+	TFOut      string
+	PulumiOut  string
```

Comment:
Can we have a doc comment here explaining what `PulumiOut` (and `TFOut`) is?

Is it stdout, stderr, both? Is it the result of `pulumi up` or `pulumi preview`?